<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>680</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>254</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="ctoaButton">
       <property name="text">
        <string>C-&gt;A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ctobButton">
       <property name="text">
        <string>C-&gt;B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btoaButton">
       <property name="text">
        <string>B-&gt;A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="atocButton">
       <property name="text">
        <string>A-&gt;C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="atobButton">
       <property name="text">
        <string>A-&gt;B</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btocButton">
       <property name="text">
        <string>B-&gt;C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>166</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDifficulty">
       <property name="text">
        <string>Difficulty:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="easyRadioButton">
       <property name="text">
        <string>Easy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="middleRadioButton">
       <property name="text">
        <string>Middle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="difficultRadioButton">
       <property name="text">
        <string>Difficult</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Best Record:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumberBestMin">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(252, 175, 62);</string>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberBestSec">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(252, 175, 62);</string>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string>Play Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdNumberMin">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 151, 137);</string>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberSec">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 151, 137);</string>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>50</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeyInput">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStartClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onResetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>easyRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onEasyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>middleRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onMiddleToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>difficultRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onDifficultToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atobButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAtoBClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atocButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAtoCClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btoaButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtoAClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btocButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtoCClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctoaButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCtoAClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctobButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCtoBClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartClicked()</slot>
  <slot>onEasyToggled(bool)</slot>
  <slot>onMiddleToggled(bool)</slot>
  <slot>onDifficultToggled(bool)</slot>
  <slot>onResetClicked()</slot>
  <slot>onAtoBClicked()</slot>
  <slot>onAtoCClicked()</slot>
  <slot>onBtoAClicked()</slot>
  <slot>onBtoCClicked()</slot>
  <slot>onCtoAClicked()</slot>
  <slot>onCtoBClicked()</slot>
 </slots>
</ui>
